geotrek/tourism/migrations/0027_touristicevent_participants.py

```diff
@@ -0,0 +1,37 @@
+# Generated by Django 3.2.15 on 2022-09-27 14:25
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('tourism', '0026_auto_20220907_1400'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='TouristicEventParticipantCategory',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('deleted', models.BooleanField(default=False, editable=False, verbose_name='Deleted')),
```

Comment:
Actually, we don't use NoDeleteMixin for categories, and we plan to remove it where it is used (treks, signages...)